src/server/index.js

```diff
@@ -32,12 +39,25 @@ function processCaption(asset, client) {
     });
 }
 
+function updatePosterFrame(asset, client) {
+  const { id: videoId, customPosterFrame, posterframeNumber } = asset.data.video;
```

Comment:
Can this name be consistent with the rest?
```suggestion
  const { id: videoId, customPosterFrame, posterFrameNumber } = asset.data.video;
```